conans/client/cache/cache.py

```diff
@@ -153,6 +169,7 @@ def config_install_file(self):
     # TODO: cache2.0 this will be removed in the future is just to adapt to some tests
     #  that call this directly
     def package_layout(self, ref, short_paths=None):
+        raise Exception("buuuuuuuhhhhh")
```

Comment:
Buuuuuhhhhhh!!! 😄 


If this is no longer used, just remove the method.

Reply:
I just pushed the new code 😉